### ISSUE TITLE
Order new batch of RITlug stickers

### ISSUE THREAD

**Issue by jwflory** [RITlug/tasks#83]
Per discussion in #67, we want to order a new batch of stickers with our club budget. This will lead up to us asking SG for financial help for a tablecloth (#35).

Next step, @jibby0 will start the EAF process to order the stickers.

**Comment by Tjzabel** [comment]
I can help with the process of choosing the actual stickers since I went through this process last Spring. I would like to change the size of the stickers we ordered as well.

**Comment by Tjzabel** [comment]
Reposting here for clarity:

```
Custom 5" x 1.88" Die cut stickers
Qty: 30
Custom 4" x 3.95" Die cut stickers
Qty: 30
Custom 2.88" x 3" Die cut stickers
Qty: 30
Subtotal: $148
```

**Comment by ct-martin** [comment]
@Tjzabel edited your comment to make it clear that's what we ordered in Spring

**Comment by jibby0** [comment]
I've been putting this off, watching Sticker Mule's sale this week. But it doesn't seem like they're offering anything we're interested in.

I'll get the EAF and the purchase submitted this week. Before the purchase, I wanted to confirm: which designs were used for which sizes? I'm guessing:

5" x 1.88" : horizontal logo
4" x 3.95": vertical logo
2.88" x 3": tigwin

**Comment by Tjzabel** [comment]
@jibby0 you are correct in your assumptions. 

When we had originally gotten these stickers, we wanted to go a little smaller with the vertical logo.
@ct-martin @jwflory what do you think?

**Comment by ct-martin** [comment]
We did. Probably 3x3" instead of 4x4" then. I marked this as "review at meeting"

**Comment by jibby0** [comment]
> The 5" x 1.88" stickers are the RITlug main logo (vertical) stickers. 

@Tjzabel I thought the 5" x 1.88" were the horizontal stickers. Just confirming, was this logo (the vertical) printed in 4" x 3.95", or 5" x 1.88"? https://github.com/RITlug/brand/blob/master/logo/png/vertical_color_no_wordmark.png

If the verticals were indeed on 4" x 3.95", I'll order them on 3" x 2.95" this time around.

**Comment by Tjzabel** [comment]
@jibby0 yep! that's the vertical image we used.

Thanks for taking care of this :+1: 
Also, how many are you thinking of ordering? How many stickers in total do we have left? I still have all mine minus a few

**Comment by jibby0** [comment]
Right now I'm looking at:

```
| Dimensions | Quantity | Image           | Cost |
|------------+----------+-----------------+------|
| 5" x 1.88" |       60 | Horizontal Logo | $72  |
| 3" x 2.95" |       60 | Vertical Logo   | $74  |
| 2.88" x 3" |       60 | Tigwin          | $72  |

Total: $160
```

@ct-martin recommended double what we ordered last year, as we went through that in a semester + summer.

**Comment by ct-martin** [comment]
To be clear though, that many should get us through the end of Spring
semester

--
Christian Martin (Phone)

**Comment by icflournoy** [comment]
### 2018-10-14 Eboard Meeting
- We were hoping there was a Sticker Mule sale. Unfortunately that time has passed.
- We are scaling down the vertical sticker (see Tim/Josh conversation above)

**Comment by jibby0** [comment]
Update:

- The EAF is all filled out.
- I forgot the sticker designs needed approval: that's been submitted, currently waiting on that.
- Part of the EAF for something like stickers requires a quote from The Hub. I'm not sure they do custom sizes, or die-cut stickers: I sent them an email asking if they did, and asking for a quote if they do. Currently waiting on a response.

**Comment by Tjzabel** [comment]
@jibby0 I would like to request that you put this on hold for the moment if possible. 

I would like to hear back from StickerMule before we finish this order.

**Comment by jibby0** [comment]
@Tjzabel Good call, we'll wait on #84. In the meantime, our designs have been approved.

**Comment by icflournoy** [comment]
### 2018-10-31 Eboard Meeting
- Blocked on discussion with StuckerMule about sponsorship
Related: https://github.com/RITlug/tasks/issues/84 for bigger picture affiliation/sponsorship showcasing on website

**Comment by jwflory** [comment]
For what it's worth, I got an out-of-office email from my contact at StickerMule. He returns from paid time off on Nov. 10th. If we don't hear back after, I can follow-up and email again.

**Comment by Tjzabel** [comment]
@jibby0 for what it's worth, I still have quite a few stickers left over in my own collection. If need be, I can mail them over to you guys.

**Comment by jrtechs** [comment]
# Eboard Meeting Notes 11-7-18

Do we want to do stickers for makerfair? Yes.
- Follow up with @Tjzabel  to see how many stickers he has and have him ship them out.
- Flory has like 5ish.
- @jrtechs has about a dozen

As a backup option we can do a rush order.

**Comment by Tjzabel** [comment]
As an update to this issue in terms of the Maker Faire, I've mailed my RITlug stickers to Justin to take with him to the fair. 

At this point, I don't think we will be able to order any new stickers to arrive before the Maker Faire, so we are going to work with what we have. With that being said, I'm going to remove the Maker Faire milestone from this issue, since it is no longer in scope.

**Comment by jwflory** [comment]
We heard back from @jdorfman this morning. To confirm, our final sticker quantity is 60 stickers each of the three varieties detailed by @jibby0 [here](https://github.com/RITlug/tasks/issues/83#issuecomment-432819780). Will update again here once we hear back.

**Comment by Tjzabel** [comment]
@jwflory yep, that's what we agreed upon :+1:

**Comment by jdorfman** [comment]
@jwflory did you account for [hexagon versions](https://www.stickermule.com/uses/hexagon-stickers)? If not we must include more store credit ;)

**Comment by jwflory** [comment]
As an update, this ticket is blocked on RITlug/ritlug.github.io#179. Once it's closed, we can move forward with closing this ticket out. Thanks for the support, @jdorfman! :tada:

**Comment by Tjzabel** [comment]
Since this is now unblocked, and we have our credits, we are able to move forward with ordering stickers.

**Comment by icflournoy** [comment]
### 2018-11-28 Eboard Meeting Notes

Agreed on doing the 60/60/60 split (above) https://github.com/RITlug/tasks/issues/83#issuecomment-432819780

**Comment by Tjzabel** [comment]
I removed the "review at meeting" label since we have agreed on what we need to get. 

I need to just go in and order the stickers.

**Comment by Tjzabel** [comment]
Since I've totally forgotten to do this, would you guys want me to order stickers, and just have them shipped to my house? Since we aren't using an RIT VISA card, we don't need to have these stickers shipped to the clubs office.

**Comment by jrtechs** [comment]
@Tjzabel I have no problem with that. How long do orders typically take to get fulfilled?

**Comment by jwflory** [comment]
@Tjzabel +1, so long as you are confident you can bring them up in time for spring semester. :slightly_smiling_face: Do eet!

**Comment by Tjzabel** [comment]
Figured I would order stickers today (Dec 30). They won't come to my house until after I leave for Rochester. Will order the stickers first thing when I'm back on campus.

**Comment by Tjzabel** [comment]
This was uh, finally done. 

I'm going to keep this issue up until the stickers are in my hands.

**Comment by Tjzabel** [comment]
#### *Small Update*

Stickers are actually coming next Monday, the 18th. After the initial ordering, I forgot to go back in and finalize the stickers. Usually it takes some time for the sticker sizes to finalize.

**Comment by Tjzabel** [comment]
Stickers are here! :tada: :tada:
